src/altinn-app-frontend/src/components/hooks/useDisplayData.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+
+export const useDisplayData = ({ formData }: any) => {
```

Comment:
We should try to not introduce new `any` types - I think we have typings for this already (`IFormData`), can you fix? 🙂